client/src/components/public/Projects.jsx

```diff
@@ -197,11 +218,17 @@ class _Projects extends Component {
               There are no {state} projects at the moment.
             </Typography>
           ) : (
-            <PublicProjectListing
-              projects={projects}
-              isLoading={this.state.isLoading}
-              projectState={projectState}
-            />
+            <React.Fragment>
+              <Pagination
+                noPages={this.state.noPages[projectState]}
```

Comment:
Let's rename the `noPages` prop of the `Pagination` component as well. Either change it to `numPages`, or follow react-paginate's prop `pageCount`